src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -101,16 +101,24 @@
                      :amount
                      (when (= transaction-type :collecible) :tx-type))})))
 
-(rf/reg-event-fx :wallet/send-select-collectible
- (fn [{:keys [db]} [{:keys [collectible stack-id]}]]
+(rf/reg-event-fx :wallet/send-collectibles-amount
+ (fn [{:keys [db]} [{:keys [collectible stack-id amount]}]]
    {:db (-> db
             (update-in [:wallet :ui :send] dissoc :token)
             (assoc-in [:wallet :ui :send :collectible] collectible)
             (assoc-in [:wallet :ui :send :tx-type] :collectible)
-            (assoc-in [:wallet :ui :send :amount] 1))
-    :fx [[:dispatch [:wallet/get-suggested-routes {:amount 1}]]
+            (assoc-in [:wallet :ui :send :amount] amount))
+    :fx [[:dispatch [:wallet/get-suggested-routes {:amount amount}]]
          [:navigate-to-within-stack [:screen/wallet.transaction-confirmation stack-id]]]}))
 
+(rf/reg-event-fx :wallet/select-collectibles-amount
+ (fn [{:keys [db]} [{:keys [collectible stack-id]}]]
+   {:db (-> db
+            (update-in [:wallet :ui :send] dissoc :token)
+            (assoc-in [:wallet :ui :send :collectible] collectible)
+            (assoc-in [:wallet :ui :send :tx-type] :collectible))
+    :fx [[:navigate-to-within-stack [:screen/wallet.select-collectible-amount stack-id]]]}))
```

Comment:
hmm, it could depend on how much work is needed on the other screens to add that code though? perhaps that is better added in a follow up issue/pr? 🤔

Reply:
we also have to add the wizard for the other parts of the collectibles sending flow. This pr is only adding 1 screen to a pre existing set of navigations.

Reply:
We can do this in a follow up issue, sure 👍 
Do we have an issue for that? Also would be nice to add docs or guidelines for using the wizard mechanism.

Reply:
Still https://github.com/status-im/status-mobile/pull/19386#discussion_r1537829259 is valid just that we can keep `:navigate-to-within-stack` (we should dispatch the event though, not use the effect).

Reply:
@J-Son89 I haven't added issues for it, since it was recently merged and I haven't started with it yet.

Seems it's very needed, I can start working on it by tomorrow